tests/hooks/test_dbapi.py

```diff
@@ -44,6 +44,7 @@ def get_conn(self):
                 return conn
 
         self.db_hook = UnitTestDbApiHook()
+        self.db_hook_log_sql = UnitTestDbApiHook(log_sql=False)
```

Comment:
```suggestion
        self.db_hook_no_log_sql = UnitTestDbApiHook(log_sql=False)
```

This should be clearer? But since this is only used by one test, it’s better to simply instantiate the hook in that test instead.

Reply:
Sure, I can rename it. I instantiate the hook here because `UnitTestDbApiHook` is defined here, I could pull it out perhaps? Also `db_hook_schema_override` is only used in one test and defined here, so I kept that pattern.

Reply:
A lot of the provider tests are historically suboptimally written, they don’t always serve as good style guides 🙂